programs/ziti-edge-tunnel/netif_driver/darwin/utun.c

```diff
@@ -283,4 +283,8 @@ netif_driver utun_open(char *error, size_t error_len, const char *cidr) {
         }
     }
     return driver;
+}
+
+const char* get_tun_name(netif_handle tun) {
```

Comment:
could this be added to the `netif_driver` interface instead of a standalone function? e.g:

netif_driver.h
```
typedef int (*netif_close_cb)(netif_handle dev);
typedef const char * (*get_name_fn)(netif_handle dev);
// ... other typedefs

typedef struct netif_driver_s {
  get_name_fn get_name;
  // ... other fields
} netif_driver_t;
```

it would then be called through the netif driver pointer that was returned by `tun_open`. e.g.:

```
const char *tun_name = tun->get_name(tun->handle);
```

this would follow the pattern for all of the other implementation-specific functions, so for instance you put all of the wide chart stuff in the windows implementaion of tun_get_name instead of ifdefing it in ziti-edge-tunnel.c